python_src/src/lamp_py/postgres/postgres_utils.py

```diff
@@ -338,55 +321,6 @@ def add_metadata_paths(self, paths: List[str]) -> None:
                 sa.insert(MetadataLog.__table__), [{"path": p} for p in paths]
             )
 
-    def seed_metadata(self) -> None:
-        """
-        seed metadata table for dev environment
-        """
-        process_logger = ProcessLogger("seed_metadata")
-        process_logger.log_start()
-        try:
-            seed_file = os.path.join(
-                HERE,
-                "..",
-                "..",
-                "..",
-                "tests",
-                "test_files",
-                "july_17_filepaths.json",
-            )
-            seed_file = os.path.abspath(seed_file)
-            with open(seed_file, "r", encoding="utf8") as seed_json:
-                paths = json.load(seed_json)
-
-            self.add_metadata_paths(paths)
-
-            process_logger.log_complete()
-        except Exception as exception:
-            process_logger.log_failure(exception)
-
-    def reset_tables(self, clear_rt: bool, clear_static: bool) -> None:
-        """
-        reset tables for dev environment
-        """
-        if clear_rt:
-            self.truncate_table(VehicleTrips)
-            self.truncate_table(VehicleEventMetrics)
-            self.truncate_table(VehicleEvents)
-
-            reset_rt_processed = (
-                sa.update(MetadataLog.__table__)
-                .values(
-                    processed=False,
-                    process_fail=False,
-                )
-                .where(MetadataLog.path.like("%RT_%"))
-            )
-            self.execute(reset_rt_processed)
-
-        if clear_static:
-            SqlBase.metadata.drop_all(self.engine)
-            SqlBase.metadata.create_all(self.engine)
-
 
```

Comment:
this is all handled in the script now. it makes the db manager _more_ agnostic of what its connecting to. (it does still have a function to get unprocessed files, which requires some info about the metadata table.)